tutorials/10-sprite-sheet/inc/fontSprite.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+# _____        ____   ___
+#   |     \/   ____| |___|
+#   |     |   |   \  |   |
+#-----------------------------------------------------------------------
+# Copyright 2022, tyra - https://github.com/h4570/tyra
+# Licensed under Apache License 2.0
+# Sandro Sobczyński <sandro.sobczynski@gmail.com>

```

Comment:
Add your name but keep sandro and replace copyright data to 2022-2023 on every file tutorial header and source file.

Reply:
Also can you do clang format?

Reply:
Well I'll change that, but what is that clamp format?

Reply:
for the tutorial you need this images 
![atlas](https://github.com/h4570/tyra/assets/104105647/91b666a2-670a-437a-93e0-a052766be14c)
![earthbound-Font](https://github.com/h4570/tyra/assets/104105647/35287ef7-a73c-44af-8561-72a7ad10b15e)